app/src/main/res/layout/fragment_login.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
remember to run a `Reformat Code` to have everything organized and to add a newline at the end of `<androidx.constraintlayout.widget.ConstraintLayout`, is just a wolox convention